Resources/Prototypes/Reagents/narcotics.yml

```diff
@@ -314,6 +314,16 @@
   physicalDesc: reagent-physical-desc-acrid
   flavor: bitter
   color: "#215263"
+  metabolisms:
+    Narcotic:
+      effects:
+      - !type:MovespeedModifier
+        walkSpeedModifier: 0.65
+        sprintSpeedModifier: 0.65
+      - !type:HealthChange
+        damage:
+          types:
+            Cellular: 4 # TODO: Make this deal brain damage instead of genetic damage once brain damage is added
```

Comment:
im pretty sure the point of brain damage is that it caused hallucinations, i dont really see the point of this as it would make Impedrezene a bad poison instead of a narcotic

Reply:
Impedrezene wasn't really a narcotic before this either because it didn't actually do anything, but i see your point.

I can add the space drug hallucination effect and/or lower the genetic damage, but full-on brain damage hallucinations do not exist yet. If they're a requirement for merging in a new effect to impedrezene then we're better off removing it.

(also, 4 genetic isn't really a "bad poison", it's a nearly unstoppable and possibly permanent death at 25u, thus the convoluted recipe)

Reply:
Just added hallucinations and replaced cellular damage with poison, so this is resolved either way..